sqlmesh/core/config/root.py

```diff
@@ -149,10 +146,16 @@ def _validate_regex_keys(
 
     @model_validator(mode="before")
     @model_validator_v1_args
-    def _normalize_fields(cls, values: t.Dict[str, t.Any]) -> t.Dict[str, t.Any]:
+    def _normalize_and_validate_fields(cls, values: t.Dict[str, t.Any]) -> t.Dict[str, t.Any]:
         if "gateways" not in values and "gateway" in values:
             values["gateways"] = values.pop("gateway")
 
+        for plan_deprecated in ("auto_categorize_changes", "include_unmodified"):
```

Comment:
this is a breaking change for users so we should do feat!

Reply:
thanks, good point